packages/devtools_app/lib/src/screens/debugger/variables.dart

```diff
@@ -176,16 +175,16 @@ class VariableSelectionControls extends MaterialTextSelectionControls {
     Offset selectionMidpoint,
     List<TextSelectionPoint> endpoints,
     TextSelectionDelegate delegate,
-    ClipboardStatusNotifier clipboardStatus,
-    Offset lastSecondaryTapDownPosition,
+    ClipboardStatusNotifier? clipboardStatus,
+    Offset? lastSecondaryTapDownPosition,
   ) {
     return _TextSelectionControlsToolbar(
       globalEditableRegion: globalEditableRegion,
       textLineHeight: textLineHeight,
       selectionMidpoint: selectionMidpoint,
       endpoints: endpoints,
       delegate: delegate,
-      clipboardStatus: clipboardStatus,
+      clipboardStatus: clipboardStatus!,
```

Comment:
either drop the null assert here or make the above parameter non nullable

Reply:
Keeping as is, we are overriding Flutter's `buildToolbar` method so we can't make the parameter non-nullable.